torch/nn/modules/rnn.py

```diff
@@ -425,10 +426,11 @@ class GRU(RNNBase):
             h_t = (1 - z_t) n_t + z_t h_{(t-1)} \\
             \end{array}
 
-    where :math:`h_t` is the hidden state at time `t`, :math:`x_t` is the hidden
-    state of the previous layer at time `t` or :math:`input_t` for the first
-    layer, and :math:`r_t`, :math:`z_t`, :math:`n_t` are the reset, input,
-    and new gates, respectively. :math:`\sigma` is the sigmoid function.
+    where :math:`h_t` is the hidden state at time `t`, :math:`x_t` is the input
+    at time `t`, :math:`h_{(t-1)}` is the hidden state of the previous layer
+    at time `t-1` or the initial hidden tate for the first layer, and :math:`r_t`,
```

Comment:
Sorry, but could you also change `the first layer` to ``at time `0` ``? Same for LSTM.

Reply:
Make sense. Done.  Thanks for your review!